src/main/java/seedu/address/model/book/Status.java

```diff
@@ -1,5 +1,5 @@
 package seedu.address.model.book;
-
+//@@author 592363789
```

Comment:
Remove. [You didn't code this.](https://github.com/CS2103JAN2018-F14-B4/main/pull/97)